src/rp2_common/pico_bootrom/include/pico/bootrom.h

```diff
@@ -1071,6 +1071,28 @@ static inline int rom_get_last_boot_type(void) {
  */
 int rom_add_flash_runtime_partition(uint32_t start_offset, uint32_t size, uint32_t permissions);
 
+/*! \brief  Pick A/B partition for a separate partition
```

Comment:
`rom_…` is the SDK function wrapper for the `pick_ab_partition` function - so I was using that to refer to it in the SDK (ie what this calls, and where more docs are), but using `pick_ab_partition` to refer to the behaviour of the underlying bootrom function

Reply:
I'm trying to mean "Pick A/B partition for an A/B partition pair that is not the one which the current binary has booted from" - ie if you have partitions `0 (A), 1 (B w/ 0), 2 (A), 3 (B w/ 2)` and have booted the binary in partition 0/1, that binary might call this to pick between the separate partitions 2/3. I wasn't sure how to word it succinctly.

Maybe "Pick A/B partition with TBYB guards" would be better?

Reply:
"Pick A/B partition from a different partition-pair" ? :man_shrugging:

Reply:
After rethinking, I think that mentioning the different partition pair doesn't make sense - because you'll usually be using the normal `pick_ab_partition` on a different partition pair too.

I think the main distinction of this function from the raw bootrom one is the TBYB guards, so I think "Pick A/B partition with TBYB guards" is probably best?